### ISSUE TITLE
[MAS4.1.2] Incorrect role is define for 'View details' control.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1350]
**Actual Result:**
Incorrect role is define for 'View details' control. 
When press Enter on the 'View details' control then new Tab will open instead of loading new content below in the same page.

**Note:** Same issue repro at -
1. URL -> Help ->Show a rich message using Adaptive cards -> New loaded content -> Set due date button -> New data loaded -> OK - Incorrect role is define for 'OK' control.
2. URL -> Help ->Hero card -> New loaded content ->Image -  Incorrect role is define for 'Image' control. Attachment - "MAS4.1.2_Incorrect role for Image".
3. URL -> Help ->Show review using  Adaptive card Button  -> New loaded content -> More Information - Incorrect role is define for More Information.
4. URL -> Help ->Show Restaurant information using  Adaptive card   -> New loaded content -> Get Direction and 'Call Restaurant' -> OK Button - Incorrect role is define for Get Direction and OK Button.
5. URL -> Help ->Start Oauth flow Button ->New Loaded content -> Sign in - Incorrect role is define for Sign in Button

**Expected Result:**
Correct role should be define for  'View details' control. 

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Type 'Help' in the input control at the bottom of the screen and hit enter (or click on send button).
4.  Content will upload, Navigate to any "Show a rich message using Adaptive cards" button by using Tab key and press Enter on it.
5. New data will upload.
6. Navigate to 'View details' control by using Tab key and Press Enter.
7. Observe that incorrect role is define for 'View details' control.

**User Impact:**
If Name,State and Role property is not defined properly for controls then the users will get confuse about the purpose of that control.

**MAS Reference:**
[MAS4.1.2 - Name,Role,Value](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge

**Attachment:**
1. [MAS4.1.2_Incorrect role define for 'Ok' and 'View details' controls.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2579485/MAS4.1.2_Incorrect.role.define.for.Ok.and.View.details.controls.zip)
2. ![MAS4 1 2_incorrect role for ok and view details](https://user-images.githubusercontent.com/41668935/48465324-77144680-e808-11e8-8fa7-c686ddab438e.png)
3. ![mas4 1 2_incorrect role for image](https://user-images.githubusercontent.com/41668935/48540074-13a91800-e8df-11e8-8121-5b24165fee2f.png)

**Comment by compulim** [comment]
This bug should belongs to Adaptive Card team.

The content is generated by Adaptive Card and it should have `<div role="link">` instead of `<div role="button">` for those tappable image (and same as `<button>`).

**Comment by compulim** [comment]
Reopening so we can confirm from the accessibility team.

**Comment by corinagum** [comment]
Confirmed as external by the accessibility team

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG4.1.2
